internal/openrpc/openrpc_schema.go

```diff
@@ -0,0 +1,1828 @@
+// Copyright 2019 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package openrpc
+
+// This file contains a string constant containing the JSON schema data for OpenRPC.
+
+// OpenRPCSchema defines the default full suite of possibly available go-ethereum RPC
+// methods.
+const OpenRPCSchema = `
```

Comment:
Why can't we return data from a json file?

Reply:
Yea, what [tzdybal said](https://github.com/multi-geth/multi-geth/pull/76#issuecomment-491833043); it's just more complicated that way... and we expect this string to be relatively extremely constant (set and forget), so extra mechanics to mediate the JSON as an external resource were expected to be of limited utility.